howtos/how-to-add-a-create_new_chat-button-in-talkjs/create-new-chat.js

```diff
@@ -0,0 +1,95 @@
+const $targetEl = document.getElementById('crud-modal');
+const modal = new Modal($targetEl)
+const appId = "<Add Your TalkJS APP ID Here>"
+
+Talk.ready.then(function () {
+    var me = new Talk.User({
+        id: '123457',
+        name: 'Alice',
+        email: 'alice@example.com',
+        photoUrl: 'https://talkjs.com/images/avatar-1.jpg',
+        welcomeMessage: 'Hey there! How are you? :-)',
+    });
+    window.talkSession = new Talk.Session({
+        appId: appId,
+        me: me,
+    });
+    var other = new Talk.User({
+        id: '654321',
+        name: 'Sebastian',
+        email: 'Sebastian@example.com',
+        photoUrl: 'https://talkjs.com/images/avatar-5.jpg',
+        welcomeMessage: 'Hey, how can I help?',
+    });
+
+    var conversation = talkSession.getOrCreateConversation(
+        Talk.oneOnOneId(me, other)
+    );
+    conversation.setParticipant(me);
+    conversation.setParticipant(other);
+
+    var inbox = talkSession.createInbox({ theme: 'Create-New-Chat-From-Conv-Header' });
+    inbox.select(conversation);
+    inbox.mount(document.getElementById('talkjs-container'));
+
+    inbox.onCustomConversationAction('createNewChat', (event) => {
+        showNewCoonversationModal()
```

Comment:
typo in "Conversation"

```suggestion
        showNewCoonversationModal()
```